src/SFA.DAS.ApprenticeCommitments.Api/SFA.DAS.ApprenticeCommitments.Api.csproj

```diff
@@ -0,0 +1,63 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+    <PropertyGroup>
+        <TargetFramework>netcoreapp3.1</TargetFramework>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <Content Remove="nlog.Debug.config" />
+      <Content Remove="nlog.Release.config" />
+    </ItemGroup>
+
+    <ItemGroup>
+      <None Include="nlog.Debug.config">
+        <IsTransformFile>true</IsTransformFile>
+        <DependentUpon>nlog.config</DependentUpon>
+      </None>
+      <None Include="nlog.Release.config">
+        <IsTransformFile>true</IsTransformFile>
+        <DependentUpon>nlog.config</DependentUpon>
+      </None>
+    </ItemGroup>
```

Comment:
this is new to me, I didn't think we did this. I haven't spotted us doing this before to the nlog files

Reply:
I haven't encountered using overide files for NLog either, but the other APIM projects do use them.